lib/index.js

```diff
@@ -558,6 +560,19 @@ Dataviz.prototype.prepare = function(){
     return this;
 };
 
+function renderDownloadButton({
+  element,
+  label = 'Download',
+  type = 'json',
+  data
+} = {}) {
+  const button = document.createElement('button');
+  button.innerText = label;
+  button.className = 'keen-dataviz-button';
+  button.onclick = event => downloadResults({ event, type, data});
```

Comment:
let's use addEventListener, so in case somebody wants to attach any custom action to this button later - it'll be possible (eg. some people may want to attach google analytics tracking on this button)